easybuild/easyconfigs/m/manta/manta-1.1.1.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::  Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::    Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::    MIT
+#
+# Notes::      This is a binary build and only works on 64-bit linux
+##
+
+easyblock = 'CmdCp'
+
+name = 'manta'
+version = '1.1.1'
+
+homepage = 'https://github.com/arq5x/lumpy-sv'
+description = """A probabilistic framework for structural variant discovery.
+"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://github.com/Illumina/manta/releases/download/v%(version)s']
+sources = ['manta-%(version)s.centos5_x86_64.tar.bz2']
+# https://github.com/Illumina/manta/releases/download/v1.1.1/manta-1.1.1.centos5_x86_64.tar.bz2
+
+skipsteps = ['build']
+
+files_to_copy = [(['bin/*'], 'bin'), 'lib', 'libexec', 'share']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['configManta.py', 'runMantaWorkflowDemo.py']],
+    'dirs': ['libexec'],
```

Comment:
I'm seeing an error in a test build here:
```
== 2017-11-04 09:01:39,843 build_log.py:158 ERROR EasyBuild crashed with an error (at easybuild/software/EasyBuild/3.4.1/lib/python2.7/site-packages/vsc_base-2.5.8-py2.7.egg/vsc/utils/exceptions.py:124 in __init__): Absolute path /home/huffmaa/.local/easybuild/software/manta/1.1.1/libexec passed to update_paths which only expects relative paths. (at easybuild/software/EasyBuild/3.4.1/lib/python2.7/site-packages/easybuild_framework-3.4.1-py2.7.egg/easybuild/tools/module_generator.py:852 in update_paths)
== 2017-11-04 09:01:39,843 easyblock.py:2159 WARNING Sanity check: loading fake module failed: 'Absolute path /home/huffmaa/.local/easybuild/software/manta/1.1.1/libexec passed to update_paths which only expects relative paths.'
== 2017-11-04 09:01:39,843 easyblock.py:2195 DEBUG Skiping RPATH sanity check
== 2017-11-04 09:01:39,844 build_log.py:158 ERROR EasyBuild crashed with an error (at easybuild/software/EasyBuild/3.4.1/lib/python2.7/site-packages/vsc_base-2.5.8-py2.7.egg/vsc/utils/exceptions.py:124 in __init__): Sanity check failed: loading fake module failed: 'Absolute path /home/huffmaa/.local/easybuild/software/manta/1.1.1/libexec passed to update_paths which only expects relative paths.' (at easybuild/software/EasyBuild/3.4.1/lib/python2.7/site-packages/easybuild_framework-3.4.1-py2.7.egg/easybuild/framework/easyblock.py:2199 in _sanity_check_step)
== 2017-11-04 09:01:39,844 easyblock.py:2669 WARNING build failed (first 300 chars): Sanity check failed: loading fake module failed: 'Absolute path /home/huffmaa/.local/easybuild/software/manta/1.1.1/libexec passed to update_paths which only expects relative paths.'
== 2017-11-04 09:01:39,844 easyblock.py:288 INFO Closing log for application name manta version 1.1.1
```
@boegel - is this an error at my end, or with the `easyconfig`?